GameServerCore/Packets/Handlers/PacketHandlerBase.cs

```diff
@@ -1,11 +1,12 @@
 ﻿using GameServerCore.Packets.Enums;
+using GameServerCore.Packets.PacketDefinitions;
 
 namespace GameServerCore.Packets.Handlers
 {
-    public abstract class PacketHandlerBase : IPacketHandler
+    public abstract class PacketHandlerBase<T> : IPacketHandler<T> where T: ICoreRequest
     {
         public abstract PacketCmd PacketType { get; }
```

Comment:
I do not think Core should be aware of packets channel and ID

Reply:
Actually these fields never used, the packets type and channel only stored in annotation in PacketReader class.